pytest/tests/contracts/infinite_loops.py

```diff
@@ -0,0 +1,64 @@
+# Spins up four nodes, deploy an smart contract to one node,
+# Call a smart contract method in another node
+import sys, time
+import base58
+import concurrent.futures
+import requests
+
+sys.path.append('lib')
+from cluster import start_cluster
+from transaction import sign_deploy_contract_tx, sign_function_call_tx
+from utils import load_binary_file, compile_rust_contract
+
+nodes = start_cluster(
+    4, 0, 1, None,
+    [["epoch_length", 10], ["block_producer_kickout_threshold", 80], ["transaction_validity_period", 10000]], {})
+
+wasm_file = compile_rust_contract(''' metadata! {
+  #[near_bindgen]
+  #[derive(Default, BorshDeserialize, BorshSerialize)]
+  pub struct InifiniteLoop {}
+}
+
+#[near_bindgen]
+impl InifiniteLoop {
+  pub fn start_loop(&self) {
+    loop {
+      env::log(b"hello world");
```

Comment:
I don't think we should `log` here, as that will make us hit the "maximum number of log messages" limit pretty soon, without burning much CPU. An empty loop seems better here.

Reply:
fixed, thanks for the suggestion!